src/test/java/org/broadinstitute/hellbender/engine/dataflow/datasources/VariantSourceTest.java

```diff
@@ -0,0 +1,165 @@
+package org.broadinstitute.hellbender.engine.dataflow.datasources;
```

Comment:
All test class names should end in either a `*UnitTest` or `*IntegrationTest` suffix.